test/promises-async-await.spec.js

```diff
@@ -30,6 +30,18 @@ describe("Promises", function () {
     assert.strictEqual(await foo(), 1);
   });
 
+  it("Should handle error in an async function", async function () {
```

Comment:
This test case is great at demonstrating that an `async` function returns a promise. I  would add another case demonstrating how an `async` function handles errors within the function. For example:

```js
async function handleError(shouldThrow) {
  try {
    if (shouldThrow) await Promise.reject('Whoops');
    else return 'no errors';
  } catch (err) {
    return 'caught error';
  }
}
}
```

Reply:
function added